test/associations_test.rb

```diff
@@ -46,5 +46,19 @@ def test_multiple_associations
       PersonSerializer.new(peggy, includes: [:colleagues, :superior])
         .associations.must_equal expected
     end
+
+    def test_association_override
+      peggy = Person.new
+      daniel = Person.new(id: 1, name: 'Daniel', occupation: 'Agent')
+      jack = Person.new(id: 4, name: 'Jack', occupation: 'Agent')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.